### DISCUSSION TITLE
실용주의 프로그래머 20주년 기념판(The Pragmatic Programmer 20th Anniversary Edition)

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#150] [General]
저희 세번째로 읽을 책으로 실용주의 프로그래머가 있는데요
이 책이 2005년, 2014년 판도 절판되서 중고로도 구하기가 힘들어서 어떻게 해야 하나 고민하던 차에
이번에 인사이트에서 20주년 기념판을 2월 24일에 출판한다고 합니다.

곧 ebook으로도 내주면 좋을 거 같긴 한데
책 읽는 시점이 4월 부터라 기대를 해보고 있습니다.

세번째 책으로 20주년 기념판으로 해 보시는 건 어떨까요?
허락보다 용서가 쉽다 작전으로
별다른 코멘트가 없으면 3월 말에 결정해서 이 책으로 하는 걸로 하겠습니다.

http://aladin.kr/p/6fZmv
![image](https://user-images.githubusercontent.com/17442457/153814087-062a307b-a0b3-48d6-b57e-f7ec25afa088.png)

**Comment by momoci99** [comment]
동! 의! 합니다

**Comment by jongfeel** [comment]
저는 ebook이 한 달 후에나 나올 줄 알았는데 생각보다 빨리 나와서 좋은 것 같습니다.
이 책이 나름 두께가 있어서 들고 다니면서 읽기가 좀 힘들거든요.

http://aladin.kr/p/jffOG
